.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Wing Development",
+	"image": "mcr.microsoft.com/devcontainers/base:jammy",
+  "features": {
+    "ghcr.io/devcontainers/features/aws-cli:1": {},
+    "ghcr.io/devcontainers/features/terraform:1": {},
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {},
+    "ghcr.io/devcontainers/features/node:1": {
+      "version": 18
+    },
+    "ghcr.io/devcontainers/features/rust:1": {},
+    "devwasm.azurecr.io/dev-wasm/dev-wasm-feature/rust-wasi:0": {}
```

Comment:
Not sure if this feature is needed. It says it installs wasmtime and the wasi-sdk, but we don't use wasmtime and wasi-sdk gets installed and used with a script as part of the install process.

Reply:
👍 Yupp you are right